docs/resources/security_monitoring_rule.md

```diff
@@ -134,6 +133,32 @@ Optional:
 - `learning_threshold` (Number) A number of occurrences after which signals are generated for values that weren't learned. Valid values are `0`, `1`. Defaults to `0`.
 
 
+<a id="nestedblock--options--third_party_rule_options"></a>
+### Nested Schema for `options.third_party_rule_options`
+
+Required:
+
+- `default_status` (String) Severity of the default rule case, when none of the third party cases match. Valid values are `info`, `low`, `medium`, `high`, `critical`.
+- `root_query` (Block List, Min: 1, Max: 10) Queries to be combined with third party case queries. Each of them can have different group by fields, to aggregate differently based on the type of alert. (see [below for nested schema](#nestedblock--options--third_party_rule_options--root_query))
+
+Optional:
+
+- `default_notifications` (List of String) Notification targets for the default rule case, when none of the third party cases match.
+- `signal_title_template` (String) A template for the signal title; if omitted, the title is generated based on the case name.
+
+<a id="nestedblock--options--third_party_rule_options--root_query"></a>
+### Nested Schema for `options.third_party_rule_options.root_query`
+
+Required:
+
+- `query` (String) Query to filter logs
```

Comment:
```suggestion
- `query` (String) Query to filter logs.
```

Reply:
Thanks! Applied in https://github.com/DataDog/terraform-provider-datadog/pull/2231/commits/1e4872ae66b9b61487da7cfbdc524101cda4f968.